### ISSUE TITLE
bcat IDeliveryCacheStorageService: 0 (CreateFileService) is not implemented

### ISSUE THREAD

**Issue by B3astTech** [Ryujinx/Ryujinx#1162]
bcat IDeliveryCacheStorageService: 0 (CreateFileService) is not implemented

## Service description:
[Switchbrew](https://switchbrew.org/wiki/BCAT_services#IDeliveryCacheStorageService) | [Reswitched](https://reswitched.github.io/SwIPC/ifaces.html#nn::bcat::detail::ipc::IDeliveryCacheStorageService(0))
```c
CreateFileService() -> object<nn::bcat::detail::ipc::IDeliveryCacheFileService>
```

## Required by:
https://github.com/Ryujinx/Ryujinx-Games-List/issues/849

**Comment by ertherod** [comment]
Implemented in #1154 no ?

**Comment by Thog** [comment]
Exactly 👍
